examples/multigpu/node_classification_sage.py

```diff
@@ -222,11 +390,56 @@ def run(proc_id, nprocs, devices, g, data, mode, num_epochs):
         use_uva,
         num_epochs,
     )
+    # After training, perform inference on the test data.
     layerwise_infer(proc_id, device, g, num_classes, test_idx, model, use_uva)
```

Comment:
Why this is named layerwise_infer? What does layerwise refer to?

Reply:
The function name `layerwise_infer` is probably due to the fact that in the model inference phase, the inference process is performed layer-wise. In graph neural networks (e.g. GraphSAGE), information is propagated through the layers of the network. 

But the naming of `layerwise_infer` is prevalent in all files related to `GraphSAGE` model, e.g. `graphsage.py`, so if I need to add an explanation of the function naming, I might need to go and add comments to all sections where `layerwise_infer` appears?